examples/recording_service/pluggable_storage/c/CMakeLists.txt

```diff
@@ -0,0 +1,135 @@
+#
+# (c) 2019 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+#  RTI grants Licensee a license to use, modify, compile, and create derivative
+#  works of the Software.  Licensee has the right to distribute object form
+#  only for use with RTI products.  The Software is provided "as is", with no
+#  warranty of any type, including any warranty for fitness for any purpose.
+#  RTI is under no obligation to maintain or support the Software.  RTI shall
+#  not be liable for any incidental or consequential damages arising out of the
+#  use or inability to use the software.
+#
+cmake_minimum_required(VERSION 3.11)
+
+project (PluggableStorage)
+
+# Find RTI Connext dependencies
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CONNEXTDDS_DIR}/resource/cmake")
+
+find_package(
+    RTIConnextDDS "6.0.0"
+    REQUIRED
+    COMPONENTS
+        core
+        routing_service
+)
+
+find_package(
+    RTICodeGenerator "3.0.0"
+    REQUIRED
+)
+
+set(GENERATED_CODE
+    "${CMAKE_CURRENT_SOURCE_DIR}/HelloMsg.c"
+    "${CMAKE_CURRENT_SOURCE_DIR}/HelloMsgPlugin.c"
+    "${CMAKE_CURRENT_SOURCE_DIR}/HelloMsgSupport.c"
+)
+
+# Generate type plug-in files for the storage type
+add_custom_command(
```

Comment:
@iblancasa , should we be using the new way to generate code? or isn't available for Galleon?

Reply:
It will be available in this repository as soon as we merge #161. The idea is to refactor the examples after that merge to use the new way to generate code.